docs/orchestrating-elastic-stack-applications/recipes.asciidoc

```diff
@@ -0,0 +1,19 @@
+:page_id: recipes
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-{page_id}"]
+= Recipes
+
+This section includes recipes that provide configuration examples for some common use cases.
+
+* link:https://github.com/elastic/cloud-on-k8s/tree/main/config/recipes/gclb[Expose Elasticsearch and Kibana using a Google Cloud Load Balancer (GCLB)]
+* link:https://github.com/elastic/cloud-on-k8s/tree/main/config/recipes/istio-gateway[Expose Elasticsearch and Kibana using Istio ingress gateway]
+* link:https://github.com/elastic/cloud-on-k8s/tree/main/config/recipes/logstash[Using Logstash with ECK]
+* link:https://github.com/elastic/cloud-on-k8s/tree/main/config/recipes/maps[Expose Elastic Maps Server and Kibana using a Kubernetes Ingress]
+* link:https://github.com/elastic/cloud-on-k8s/tree/main/config/recipes/psp[Secure your cluster with Pod Security Policies]
+* link:https://github.com/elastic/cloud-on-k8s/tree/main/config/recipes/traefik[Use Traefik to expose Elastic Stack applications]
+
+WARNING: Compared to other configuration examples that are consistently tested, like <<{p}-elastic-agent-fleet-configuration-examples,fleet-managed Elastic Agent on ECK>>, <<{p}-elastic-agent-configuration-examples,standalone Elastic Agent on ECK>>, or <<{p}-beat-configuration-examples,Beats on ECK>>, the recipes in this section are not regularly tested by our automation system, and therefore should not be considered to be production-ready. 
```

Comment:
Sorry but I only now got around to take a look at this. I think the wording around production readiness is misleading. The Agent and Beats recipes are also not production ready as they take certain liberties to illustrate typical configurations (e.g. no volume configuration, no memory mapped files in Elasticsearch to name just two).  The main difference is that those Beats and Agent recipes are guaranteed to work when applied to a Kubernetes cluster while the ones listed here only worked when we created them but we do not continuously test them to ensure that they still work on for example newer versions of Kubernetes.